network-api/networkapi/wagtailpages/templates/wagtailpages/research_landing_page.html

```diff
@@ -1,64 +1,92 @@
 {% extends "./research_base.html" %}
-{% load wagtailcore_tags %}
+{% load i18n wagtailcore_tags wagtailimages_tags static %}
 
 {% block subcontent %}
 
-<div class="tw-border-4 tw-border-gray-60 tw-my-5 tw-p-6">
-  <h2>
-    Page Title: {{ page.title }}
-  </h2>
-</div>
+<div class="landing-page-container">
 
-<div class="tw-border-4 tw-border-gray-60 tw-my-5 tw-p-6">
-  <h2>
-    Page Intro: {{ page.intro }}
-  </h2>
-</div>
+  {% with hero_image_classes="landing-page-hero-image tw-bg-no-repeat tw-bg-left-top tw-bg-cover tw-min-h-[260px]" %}
+    {% if page.banner_image %}
+      {% image page.banner_image original as hero_img %}
+      <div class="{{hero_image_classes}}" style='background-image: url("{{ hero_img.url }}")'></div>
+    {% else %}
+      <div  class="{{hero_image_classes}}" style='background-image: url("{% static "_images/banner-black-white-marble.jpg" %}")'></div>
+    {% endif %}
+  {% endwith %}
 
-<div class="tw-border-4 tw-border-gray-60 tw-my-5 tw-p-6">
-  <div>
-    <h2>
-      Search Bar:
-    </h2>
-    <form action="library" method="get" accept-charset="utf-8">
-      <input type="text" value="{{ search_query }}" name="search" id="search"
-        class="tw-inline-block form-control tw-w-1/2 tw-rounded-none" />
-      <button type="submit" class="tw-btn-primary">Search</button>
-    </form>
-  </div>
-</div>
+  <div class="landing-page-content tw-flex tw-w-full">
 
-<div class="tw-border-4 tw-border-gray-60 tw-my-5 tw-p-6">
-  <div>
-    <h2>
-      Browse All Button:
-    </h2>
-    <a href="{% pageurl library_page %}">Browse all</a>
-  </div>
-</div>
+    <div class="left-spacer-and-cutout tw-bg-white tw-flex-auto -tw-mt-[125px] tw-pl-4 small:tw-pl-0"></div>
 
-<div class="tw-container tw-border-4 tw-border-gray-60 tw-my-5 tw-p-6">
-  <h2>Latest Research</h2>
-  <ul class="tw-list-none tw-px-0 tw-space-y-3">
-    {% for research_detail_page in latest_research_detail_pages %}
-    <li class="tw-p-5 tw-bg-gray-05">
-      {% include "wagtailpages/fragments/research_detail_card.html" with research_detail_page=research_detail_page heading_level=3 hide_topics_on_mobile=True %}
-    </li>
-    {% endfor %}
-  </ul>
-</div>
+    <div class="content-and-cutout tw-grid tw-grid-cols-12 tw-container tw-mr-4 tw-px-0 small:tw-mr-0">
+        <div class="cutout tw-bg-white tw-col-span-12 -tw-mt-[125px] medium:tw-px-4 large:tw-col-span-8">
+          <div class="tw-pt-5 tw-mr-4 medium:tw-mr-0 large:tw-w-full">
+            <h1>{{page.title}}</h1>
+            {% if page.intro %}
+              <p class="tw-body-large mt-3 mb-0">{{ page.intro }}</p>
+            {% endif %}
+            <div class="search-bar tw-my-5 tw-w-full">
+              <form action="library" method="get" accept-charset="utf-8" class="tw-w-full medium:tw-inline-flex">
```

Comment:
**Issue** We should not be hard coding URLs like this. This would not work when the library has a different URL (the slug is not fixed, so different URLs for the library are possible). Also, this might be an issue when dealing with the different locales. E.g. a different locale might use a different slug, because it has different page title.

Reply:
Thanks for flagging this TIbor! updated to use `{% pageurl library_page %}`